### PR TITLE
Config cleanup and azure compatibility update.

### PR DESCRIPTION
**Pull request by merqlove**
Source code version upgrade to 0.7 for Azure support.
Added readme notice.
Hide Swift vars if nothing.

### CODE REVIEW

File: templates/default/environment.erb [hunk 1]
```diff
@@ -7,10 +7,12 @@
 # The duplicity password
 export PASSPHRASE="<%= @passphrase %>"
 
+<% if @swift_username && @swift_password && @swift_authurl %>
```
Comment (chr4) [inline templates/default/environment.erb]: Why is this needed? Is Azure incompatible with Switft?

Comment (merqlove) [inline templates/default/environment.erb]: This is cumulative update.

--
Best regards,
Alexander Merkulov

E-mail: sasha@merqlove.ru
Tel.:  +7(925) 278-44-98
Skype:  merqlove

Projects: 
mrcr.ru - Разработка Web-сайтов, Поисковая оптимизация

Comment (chr4) [inline templates/default/environment.erb]: You're right, this is analog to the other variables. Good catch!

**Comment by chr4** [comment]
Released in v1.2.1. Thanks again!

**Comment by merqlove** [comment]
Great!
